cpp/models/abm/location.h

```diff
@@ -254,6 +272,7 @@ class Location
     int m_num_persons = 0;
     LocationCapacity m_capacity;
     bool m_capacity_adapted_transmission_risk;
+    std::map<TimePoint, std::array<int, size_t(InfectionState::Count)>> m_timestep_to_subpopulations_map; 
```

Comment:
Please see https://github.com/DLR-SC/memilio/blob/main/cpp/memilio/compartments/simulation.h or in precise usage https://github.com/DLR-SC/memilio/blob/main/cpp/examples/ode_secir_ageres.cpp#L91 where one ODE simulation returns a TimeSeries<double>. Also the global ABM world has a a global TimeSeries already: https://github.com/DLR-SC/memilio/blob/main/cpp/models/abm/simulation.h#L91

We should have one time series per location as TimeSeries already has a lot of functionality.